src/kvstore/raftex/RaftPart.cpp

```diff
@@ -1527,10 +1527,10 @@ void RaftPart::processAppendLogRequest(
                   << ", my term is " << term_
                   << ", to make the cluster stable i will follow the high term"
                   << " candidate and clenaup my data";
-        reset();
         resp.set_committed_log_id(committedLogId_);
         resp.set_last_log_id(lastLogId_);
         resp.set_last_log_term(lastLogTerm_);
+        return;
```

Comment:
![image](https://user-images.githubusercontent.com/13706157/90469327-1dfae480-e14b-11ea-8365-da7d332775b8.png)
I only have this log, crashed in v1.0.1. I think it is the only way that lastLogId in wal is 0. PS: I am not pretty sure about whether we could `reset` in this case/